### PR TITLE
Add (truncated) basis computation of the coordinate ring of a fixed Kodaira embedding

### PR DESCRIPTION
**Pull request by lgoettgens**
Some follow-up features to the book chapter by G. Fourier and X. Fang.
I paid attention to not changing any printing of existing functionality, to not creating any issues with the book.

This PR is NOT needed for the book.

@gfourier Please have a look at the new docstrings and the tests. The rest is unchanged compared to our discussion earlier today.

### CODE REVIEW

**Review by gfourier — APPROVED** [review]

**Review by gfourier — APPROVED** [review]

**Review by gfourier — APPROVED** [review]

**Review by gfourier — APPROVED** [review]

**Comment by lgoettgens** [comment]
@gforurier you already approved 4 times 😂

**Comment by gfourier** [comment]
I wanted to be really, really sure (and lost the connection several time while on the train)

Von: Lars Göttgens ***@***.***>
Gesendet: Mittwoch, 28. Februar 2024 17:07
An: oscar-system/Oscar.jl ***@***.***>
Cc: Fourier, Ghislain ***@***.***>; Mention ***@***.***>
Betreff: Re: [oscar-system/Oscar.jl] Add (truncated) basis computation of the coordinate ring of a fixed Kodaira embedding (PR #3456)


@gforurier you already approved 4 times 😂

File: experimental/BasisLieHighestWeight/src/MainAlgorithm.jl [hunk 1]
```diff
@@ -73,9 +72,116 @@ function basis_lie_highest_weight_compute(
...
+  Pseudocode:
+
+  basis_coordinate_ring_kodaira_compute(highest_weight, degree)
+
+  returns all multiples of the given highest_weight up to degree 
+  such that for this ordering the monomial basis is not a Minkowski sum of bases of smaller multiples.
+      return monomial bases for highest_weight and for each multiple up to degree, that is not the Minkowski sum 
+        of smaller multiples, the missing monomials
+
+  """
```
Comment (thofma) [inline experimental/BasisLieHighestWeight/src/MainAlgorithm.jl]: Maybe turn this into a comment?

Comment (lgoettgens) [reply experimental/BasisLieHighestWeight/src/MainAlgorithm.jl]: Done

**Review by thofma — APPROVED** [review]

**Pull request merged**